index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>Vite App</title>
+  </head>
+  <body>
+   <header><h1>🍔햄식이를 도와줘🍔</h1></header>
+  </body>
+  <main>
+    <aside class="sideBar">
+      <h2 class="sideBar__title">다른거 먹자</h2>
+      <div class="sideBar__btn">
+      <div class="sideBar__activeBtn">맥도날드</div>
+      <div class="sideBar__inactiveBtn">버거킹</div>
```

Comment:
```suggestion
      <button class="sidebar__btn--active">맥도날드</button>
      <button class="sidebar__btn--inactive">버거킹</button>
```

BEM에서 Modifier는 Block or Element의 모양(color, size), 상태(disabled, checked, active)를 정의해요!
BEM 적용해서 이렇게 클래스명 짓는 거 추천드려요 😉

Reply:
@100Gyeon 클래스명에 대해선 아직 어렵네요,, 배워갑니다 😂